### PR TITLE
Update packaging.yaml

### PR DESCRIPTION
**Pull request by tjohns**
- Set status to PUBLISHED
- Remove "Getting Started" category, since this category is overused.
  (If needed, make a "NDK Getting Started" category?)

### CODE REVIEW

**Comment by proppy** [comment]
Ah, I had also categories change in #63, I will drop them. Can you do s/Native/NDK as well?

File: hello-jni/.google/packaging.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-status:       DRAFT
+status:       PUBLISHED
 technologies: [Android, NDK]
-categories:   [Getting Started, Native]
+categories:   [Native]
```
Comment (proppy) [inline hello-jni/.google/packaging.yaml]: s/Native/NDK

Comment (proppy) [inline hello-jni/.google/packaging.yaml]: Can you also add 'C++'?

Comment (tjohns) [inline hello-jni/.google/packaging.yaml]: Done!

**Comment by proppy** [comment]
LGTM

**Pull request merged**